lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,9 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section aria-labelledby="course-updates-heading">
```

Comment:
I am digging into this more now as I understand the goal of the changes better.  I will update when I have more detail.